zigpy/device.py

```diff
@@ -338,13 +338,26 @@ def __getitem__(self, key):
         return self.endpoints[key]
 
     def get_signature(self):
-        signature = {}
+        # return the device signature by providing essential device information
+        #    - Model Identifier ( Attribut 0x0005 of Basic Cluster 0x0000 )
+        #    - Manufacturer Name ( Attribut 0x0004 of Basic Cluster 0x0000 )
+        #    - Endpoint list
+        #        - Profile Id, Device Id, Cluster Out, Cluster In
+        signature = {"manufacturer_name": '', "manufacture_code": '', "model": ''}
+        if self._manufacturer is not None:
+            signature["manufacturer_name"] = self.manufacturer
+        if self.manufacturer_id is not None:
+            signature["manufacture_code"] = self.manufacturer_id
```

Comment:
Instead of manufacturer_id is could be better to store the node descriptor, as it gives you more information about the device

Reply:
Good point, I'll see how to put those information too